test/e2e/config/gcp-ci.yaml

```diff
@@ -41,8 +41,8 @@ providers:
   - name: kubeadm
     type: ControlPlaneProvider
     versions:
-      - name: v1.5.3
-        value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.5.3/control-plane-components.yaml
+      - name: v1.6.1
+        value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.6.1/control-plane-components.yaml
```

Comment:
bump the line 56 as well to

```
   - name: v1.6.99 # next; use manifest from source files
```

Reply:
I had it like that, but had to change it to `v1.5.99` the E2Es where erroring saying `v1.6.99` was not found.